first_timers/first_timers.py

```diff
@@ -9,15 +9,14 @@
 # Set up logging
 logging.basicConfig(filename='app.log', level=logging.INFO)
 
-
 DAYS_OLD = 15
 MAX_TWEETS_LEN = 280
+ELLIPSE = u'…'
 
-ellipse = u'…'
-api = 'https://api.github.com/search/issues'
-FIRST_ISSUE_QUERY_URL = api + '?q=label:"{}"+is:issue+is:open&sort=updated&order=desc'
+API = 'https://api.github.com/search/issues'
+FIRST_ISSUE_QUERY_URL = API + '?q=label:"{}"+is:issue+is:open&sort=updated&order=desc'
 
-# Logging helper function
+# Logging helper functions
 def log_info(message):
```

Comment:
I think we should get rid of the log comment. I do not see any value from the comment.

Reply:
ohh okay

Reply:
I don't think this is resolved.